app/Filament/Resources/GameResource/RelationManagers/MemoryNotesRelationManager.php

```diff
@@ -149,6 +149,7 @@ public function table(Table $table): Table
 
                             $memoryNote->user_id = $user->id;
                             $memoryNote->forceFill(['body' => '']);
+                            $memoryNote->deleted_at = now();
```

Comment:
The non-manage page doesn't actually blank out the note when setting the `deleted_at` field. We probably shouldn't here either.

https://github.com/RetroAchievements/RAWeb/blob/6c1922f05be87899082aa2c10984edb273d0be81/app/Connect/Actions/SubmitCodeNoteAction.php#L54-L62

Reply:
I'd suggest using the `SubmitCodeNoteAction` class, but it would take the note properties as input and have to look up the note again just to do the permissions check and delete call.

Reply:
Updated this in latest to use `DeleteAction` instead of a generic `Action` and removed the action method.

`DeleteAction` automagically knows to soft delete the record.